frontend/styles/globals.css

```diff
@@ -84,8 +84,37 @@ strong {
     font-weight: 600;
 }
 
+input,
+textarea {
+    padding: 0.25rem;
+    border: 1px solid var(--red);
+    border-radius: var(--global-radius);
+}
+
+input[type='submit'] {
+    margin-top: 1.5rem;
+    cursor: pointer;
+    background-color: var(--red);
+    color: var(--light);
+    border-radius: var(--global-radius);
+    border: 1px solid var(--red);
+    transition: all 0.5s ease;
+}
+
+input[type='submit']:disabled {
+    background-color: var(--light);
+    color: var(--darkblue);
+    border-color: var(--darkblue);
+    cursor: wait;
+}
```

Comment:
That was my idea at first but the styles didn't have effect and I was not able to put a raw input {} outside the global styles, I'll search how to do that

Reply:
oh so instead of using an element selector like `input` you would use a class...
```
// Input.module.css
.localInput[type='submit']:disabled {
....
}
```
Then in the component....
```
// Input.jsx
import classes from './Input.module.css';
<input className={classes.localInput} .... />
```

Reply:
The issue you where probably running into was using element selector in a css module file

Reply:
Exactly!! Thanks for this, just changed it at 5f5d1ff